site/learn/expert-advice/authentication/webauthn-explained.md

```diff
@@ -0,0 +1,478 @@
+
+Alice sends the public key over to Bob. She can publish it on a website, send it over email, or print it out and use a carrier pigeon, it doesn't matter. The public key is public.
+
+Next, Alice doesn't just create the message M, she creates a signature as well, which is based on mathematical operation with the private key.
+
+{% include _image.liquid src="/assets/img/advice/webauthn-explained/alice-signs-message.png" alt="Alice signs the message with her private key." class="img-fluid" figure=false %}
+
+Bob can use the public key and verify the signature matches. Because of the nature of asymmetric signatures,  only the holder of the private key could have created the signature. 
+
+{% include _image.liquid src="/assets/img/advice/webauthn-explained/bob-checks-message-signature.png" alt="Bob verifies message matches signature." class="img-fluid" figure=false %}
+
+If Charlie gets in the middle and tries to modify the message, he can (what!). But he can't change the signature because only the holder of the private key can create the correct signature.
+
+{% include _image.liquid src="/assets/img/advice/webauthn-explained/charlie-tries-to-modify-message-with-signature.png" alt="Charlie can modify the message, but not the signature." class="img-fluid" figure=false %}
+
+When Bob checks the signature that arrived with the modified message, it fails. Bob then knows the message has been tampered with.
+
+{% include _image.liquid src="/assets/img/advice/webauthn-explained/signature-doesnt-validate.png" alt="The signature is not valid because the message contents have changed." class="img-fluid" figure=false %}
+
+This is a brief overview of public/private key signatures. The key takeaway is that a signature ensures the validity of the message received. 
+
+## Standards
+
+Taking a step back, WebAuthn was standardized by a partnership between FIDO and the W3C. FIDO is an organization that's been around for years. It stands for Fast Identity Online and it creates standards focused on passwordless authentication. FIDO2 is the second version of the standard and WebAuthn is part of FIDO2. It was standardized starting in 2015 and fully baked in 2019.
+
+WebAuthn focuses on the browser interaction with a passwordless protocol. CTAP2 is a complementary standard which handles communication between the browser and the hardware or software authenticator. (The yellow smiley face below is the user and the white globe is the browser.)
+
+{% include _image.liquid src="/assets/img/advice/webauthn-explained/ctap2-webauthn.png" alt="Showing the difference between CTAP2 and WebAuthn." class="img-fluid" figure=false %}
+
+If you have a website that lets a user log in with an authenticator, then CTAP2 is handling communication between the authenticator and the browser. 
+
+You can use CTAP2 without WebAuthn, for example, with a native app.
```

Comment:
This may be true, but the APIs I've found for Android and iOS more closely mirror the WebAuthn API than CTAP2.
- [Supporting passkeys on Apple devices](https://developer.apple.com/documentation/authenticationservices/public-private_key_authentication/supporting_passkeys)
- [FIDO2 API for Android](https://developers.google.com/identity/fido/android/native-apps)